app/src/main/java/org/feature/fox/coffee_counter/data/local/database/dao/UserDao.kt

```diff
@@ -41,4 +43,15 @@ interface UserDao {
 
     @Delete
     suspend fun deleteImageDb(image: Image)
+
+    @Transaction
+    @Query("SELECT * FROM purchase WHERE userId=:userId")
+    suspend fun getPurchaseListOfUserDb(userId: String): List<Purchase>
+
+    @Transaction
+    @Query("SELECT * FROM achievement WHERE userId=:userId")
+    suspend fun getAchievementListOfUserDb(userId: String): List<Achievement>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    suspend fun insertAchievementDb(achievement: Achievement)
```

Comment:
Move and sort queries so it is easier to navigate the UserDao: `inserts -> updates -> querys -> deletes`.